### PR TITLE
Add disclaimer about anonymous surveys

### PR DESCRIPTION
**Pull request by jonasdeluna**
image

### CODE REVIEW

**Review by eikhr — APPROVED** [review]
Looks great!

**Review by LudvigHz — APPROVED** [review]
LGTM :shipit:

File: app/routes/surveys/components/SubmissionEditor/SubmissionEditor.js [hunk 1]
```diff
@@ -40,6 +40,9 @@ const SubmissionEditor = ({
         Spørreundersøkelse for arrangementet{' '}
         <Link to={`/events/${survey.event.id}`}>{survey.event.title}</Link>
       </div>
+      <div className={styles.surveyTime}>
```
Comment (ivarnakken) [inline app/routes/surveys/components/SubmissionEditor/SubmissionEditor.js:43]: What a retarded class name - it only adds color and has very little to do with time ... oh well 🤷🏼

**Review by ivarnakken — APPROVED** [review]
💯 Awesome

**Pull request merged**